### ISSUE TITLE
Pulmonary artery adventitial fibrosis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6522]
**Preferred term label:** Pulmonary artery adventitial fibrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Accumulation of the fibroblasts in the adventitia PMID: 23737168

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Pulmonary arterial hypertension associated with congenital heart disease

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
I think a better definition is

Increased amount of collagen, proteoglycans, and other extracellaulr matrix proteins resulting in scarring and thickening in the adventia of the pulmonary artery. (Question -- do we actually mean the pulmonary arterial tree rather than just the pulmonary artery sensu strictu? Should we cite this article for additional context: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6039811/).

**Comment by eswietlik** [comment]
Agree, sorry, was rushing through it and did not finish the definition.
Yes, the adventitial fibrosis affects a significant proportion of pulmonary arterial tree (rather than just main, R and L PAs) and plays a significant role in disease pathogenesis PMID: 17008597; PMID: 23737168

**Comment by pnrobinson** [comment]
adding term, thanks!
